remote/remote_state_s3.go

```diff
@@ -79,22 +79,29 @@ var terragruntOnlyConfigs = []string{
 	"bucket_sse_kms_key_id",
 }
 
+type RemoteStateConfigS3AssumeRole struct {
+	RoleArn     string `mapstructure:"role_arn"`
+	ExternalID  string `mapstructure:"external_id"`
+	SessionName string `mapstructure:"session_name"`
+}
+
 // A representation of the configuration options available for S3 remote state
 type RemoteStateConfigS3 struct {
-	Encrypt          bool   `mapstructure:"encrypt"`
```

Comment:
It seems that aside from your changes, the text was also reformatted ...

Reply:
I think it's just the alignment from `go fmt` because of the new type `RemoteStateConfigS3AssumeRole`